FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/History.txt

```diff
@@ -1,3 +1,47 @@
+Changes between V2.2.1 and V2.2.2
+	+ Add CBMC proofs for 4 functions.
+	+ Address ~1200 MISRA violations.
+	+ Support caching multiple IP addresses per DNS entry in TCP library.
+	+ Security improvements in the TCP library.
```

Comment:
Is this in addition to the CBMC work?  Again I would make it either more generic as "continuous security improvements" or more specific such as "Improved security by giving users the option to reject UDP packets with a checksum of zero even though the spec allows it".